llvm/test/CodeGen/AArch64/aarch64-combine-add-zext.ll

```diff
@@ -0,0 +1,20 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -mtriple=aarch64-none-linux-gnu | FileCheck %s
+
+define i32 @test_add_zext(<4 x i16> %a, <4 x i16> %b) local_unnamed_addr #0 {
```

Comment:
It is probably worth adding other type sizes if they are not already tested elsewhere.

Reply:
I've added more types to this test, but I'm wondering if it may be redundant? The tests in vecreduce-add.ll seem to cover those cases as well.